### DISCUSSION TITLE
[Enhancement] [select] select 能否增加tag最多显示数量api

### DISCUSSION THREAD

**Discussion by biubiubiu01** [element-plus/element-plus#10809] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

### 具体描述
- multiple 多选时添加 tag slot插槽
or
- collapseTags模式下增加maxTag显示数量，超过这个数量后再折起。

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
已经有此功能
